submissions/asaMitaka/frogger/app.js

```diff
@@ -0,0 +1,169 @@
+// Enemies our player must avoid
+var newP = document.createElement('p');
+newP.textContent = `Your score: 0 `;
+document.body.append(newP);
+
+var enemyStat = {
```

Comment:
There is no reason for a separate object that stores information about enemies since you already has a whole class for them. Store these values there.